docs/_docs/2_3_querying_data.md

```diff
@@ -9,7 +9,7 @@ last_modified_at: 2019-12-30T15:59:57-04:00
 
 Conceptually, Hudi stores data physically once on DFS, while providing 3 different ways of querying, as explained [before](/docs/concepts.html#query-types). 
 Once the table is synced to the Hive metastore, it provides external Hive tables backed by Hudi's custom inputformats. Once the proper hudi
-bundle has been provided, the table can be queried by popular query engines like Hive, Spark and Presto.
+bundle has been provided, the table can be queried by popular query engines like Hive, Spark datasource, Spark SQL and Presto.
```

Comment:
should we also mention the impala?

Reply:
it's not released yet though..but we should file a ticket for doc-ing that nonetheless.

Reply:
+1 Created https://issues.apache.org/jira/browse/HUDI-630